spec/views/api/guidance/show.html.erb_spec.rb

```diff
@@ -12,25 +12,4 @@
     header_text = "Lead provider guidance: early career training programme"
     expect(view.content_for(:page_header)).to have_text(header_text)
   end
-
-  context "when a release note is present" do
```

Comment:
not sure why this was deleted? I think the main guidance page will still show the release note?

Reply:
ah yes, will undo this chnage